.github/workflows/ci.yml

```diff
@@ -49,7 +49,10 @@ jobs:
     - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # 2.4.0
 
     - name: Download all `exercism/nim` exercises
-      run: git clone --depth 1 https://github.com/exercism/nim.git
+      run: |
+        git clone https://github.com/exercism/nim.git
+        cd nim
+        git checkout 8710920
```

Comment:
I assume this is temporary? If so, maybe add a comment explaining why this is being done?

Reply:
Ya, that is what I meant by the first comment on the PR.

There are 2 commits. The first one is to specifically checkout my commit that adds tests for the lasagna exercise and the second adds the concept exercises to the docker container tests. Without the a non-empty test file, the test runner errors out (I'm not 100% percent sure, but I'm pretty sure we insert some unit test output formatter setup code before the first test suite, but there is no test suite so we attempt to close the formatter file stream which errors out), thus making the CI fail.

Reply:
Updated description

Reply:
Okay, great

Reply:
Commit removed as https://github.com/exercism/nim/pull/339 has been merged